examples/nidcpower_source_dc_voltage_with_multiplexer/pyproject.toml

```diff
@@ -0,0 +1,36 @@
+[tool.poetry]
+name = "nidcpower_source_dc_voltage"
+version = "0.5.0"
+description = "MeasurementLink example that sources and measures a DC voltage with an NI SMU connected via an NI-SWITCH multiplexer."
+authors = ["National Instruments"]
+
+[tool.poetry.dependencies]
+python = "^3.8"
+nidcpower = { version = ">=1.4.4", extras = ["grpc"] }
+niswitch = { version = ">=1.4.4", extras = ["grpc"] }
+ni-measurementlink-service = {version = "^1.4.0"}
+click = ">=7.1.2, !=8.1.4" # mypy fails with click 8.1.4: https://github.com/pallets/click/issues/2558
+grpcio = "*"
+
+[tool.poetry.group.dev.dependencies]
+ni-python-styleguide = ">=0.4.1"
+mypy = ">=1.0"
+grpc-stubs = "^1.53"
+# Uncomment to use prerelease dependencies.
+# nidcpower = { git = "https://github.com/ni/nimi-python.git", subdirectory = "generated/nidcpower"}
+ni-measurementlink-service = {path = "../..", develop = true}
```

Comment:
I'll comment this line before I check in the PR changes.

Reply:
You need a pre-release or release build incorporating your other changes before you can comment this out.

It's fine to leave it uncommented until the next pre-release. See https://github.com/ni/measurementlink-python/wiki/Release-Process#example-versioning